CanHazFunny/CanHazFunny/Program.cs

```diff
@@ -1,10 +1,20 @@
-﻿namespace CanHazFunny;
+using System;
 
-class Program
+namespace CanHazFunny
 {
-    static void Main(string[] args)
+    class Program
     {
-        //Feel free to use your own setup here - this is just provided as an example
-        //new Jester(new SomeReallyCoolOutputClass(), new SomeJokeServiceClass()).TellJoke();
+        static void Main(string[] args)
```

Comment:
```suggestion
        public static void Main(string[] args)
```
same accessibility requirement as above